src/api/app/models/group.rb

```diff
@@ -7,7 +7,7 @@
 class Group < ApplicationRecord
   has_one :staging_workflow, class_name: 'Staging::Workflow', foreign_key: :managers_group_id, dependent: :nullify
   has_many :groups_users, inverse_of: :group, dependent: :destroy
-  has_many :users, -> { distinct }, through: :groups_users
+  has_many :users, -> { distinct.order(:login) }, through: :groups_users
```

Comment:
nice solution! :+1: It is also affecting the old UI, but I think it is ok

Reply:
btw, `includes(:users)` (instead of `includes(:groups_users, :users)`) in the controller is enough. (although not necessarily part of this PR)

Reply:
Good catch, I will fix it.